.gitignore

```diff
@@ -2,4 +2,5 @@ phpunit.xml
 composer.lock
 build
 vendor
-coverage.clover
\ No newline at end of file
+coverage.clover
+ 
```

Comment:
Yeah, that's what SensionLabs Insight is looking for. I know, it's silly.... using Fabien's php-cs-fixer gets this fixed too.